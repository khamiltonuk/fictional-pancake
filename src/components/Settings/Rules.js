// @flow
import React, { Fragment } from "react";
import styled from "styled-components";

const Title = styled.h2`
  font-size: 100%;
  font-weight: normal;
`;

const Rules = () => {
  return (
    <Fragment>
      <Title>Rules</Title>
      <p>
        "It's very simple. Scissors cuts paper, paper covers rock, rock crushes
        lizard, lizard poisons Spock, Spock smashes scissors, scissors
        decapitates lizard, lizard eats paper, paper disproves Spock, Spock
        vaporizes rock, and as it always has rock crushes scissors"
      </p>
    </Fragment>
  );
};

export default Rules;
