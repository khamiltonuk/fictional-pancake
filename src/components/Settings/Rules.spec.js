import React from "react";
import { shallow } from "enzyme";
import Rules from "./Rules";
import toJson from "enzyme-to-json";

describe("Rules", () => {
  test("should match the snapshot", () => {
    const wrapper = shallow(<Rules />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
