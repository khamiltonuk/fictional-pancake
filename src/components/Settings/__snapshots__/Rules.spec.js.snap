// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rules should match the snapshot 1`] = `
<React.Fragment>
  <styled.h2>
    Rules
  </styled.h2>
  <p>
    "It's very simple. Scissors cuts paper, paper covers rock, rock crushes lizard, lizard poisons Spock, Spock smashes scissors, scissors decapitates lizard, lizard eats paper, paper disproves Spock, Spock vaporizes rock, and as it always has rock crushes scissors"
  </p>
</React.Fragment>
`;
